menuinst/platforms/base.py

```diff
@@ -22,6 +22,7 @@
 )
 
 log = getLogger(__name__)
+SCHEMA_VERSION = "1-1-0"
```

Comment:
Same here, should we detect the latest version here instead of hard-coding it in two places?

Reply:
I don't think we should here, because it's a runtime bit of the package.